robot_smach_states/src/robot_smach_states/navigation/navigate_to_grasp.py

```diff
@@ -64,6 +64,7 @@ def generateConstraint(self):
             rz = 0
 
         # Outer radius
+        radius -= 0.1
```

Comment:
Why?

Reply:
"Stand more close to place and grasph, RRT fails too often" according to @reinzor